src/vm/eventpipe.cpp

```diff
@@ -289,6 +395,34 @@ void EventPipe::WriteEvent(EventPipeEvent &event, BYTE *pData, unsigned int leng
         NOTHROW;
         GC_NOTRIGGER;
         MODE_ANY;
+    }
+    CONTRACTL_END;
+
+    EventPipeEventPayload payload(pData, length);
+    EventPipe::WriteEventInternal(event, payload, pActivityId, pRelatedActivityId);
+}
+
+void EventPipe::WriteEventBlob(EventPipeEvent &event, EventData **pBlobs, unsigned int blobCount, LPCGUID pActivityId, LPCGUID pRelatedActivityId)
```

Comment:
Addressed in https://github.com/dotnet/coreclr/pull/13347/commits/851943c5d5dc653dc1625a376c26bc2d6cf6a587